sos/plugins/numa.py

```diff
@@ -39,7 +39,8 @@ def setup(self):
         self.add_copy_spec([
             os.path.join(numa_path, "node*/meminfo"),
             os.path.join(numa_path, "node*/cpulist"),
-            os.path.join(numa_path, "node*/distance")
+            os.path.join(numa_path, "node*/distance"),
+            os.path.join(numa_path, "node*/hugepages/hugepages-*/*"),
```

Comment:
The trailing comma is technically fine, but it's not a style used in sos. Stylistically, the last element in a list does not have a comma for us.